sdk/core/azure-core/tests/test_utils.py

```diff
@@ -146,3 +146,28 @@ def test_parse_retry_after():
     assert ret == 0
     ret = parse_retry_after("0.9")
     assert ret == 0.9
+
+
+def test_sanitize_url_query_params():
```

Comment:
I'd like to see more test cases. Examples: multiple instances of the same key. Empty value, REDACTED as the parameter name. Query parameter in the path. (e.g. https://a.b.com/hello/world=true/something).